### DISCUSSION TITLE
[SkyBlock] Add Pets menu

### DISCUSSION THREAD

**Discussion by LeaPhant** [HypixelDev/PublicAPI#185] [Ideas]
Hey,

The Pets update has been released a few days ago and as of now there's no info about them in the API. It should include a list of all pets in someone's pet menu + which pet they currently have summoned. NBT data of each pet as it is visible in the game should be enough. :)

**Comment by LeaPhant** [comment]
Pets have been added to the API but only their id, xp and rarity. I still need the pets' NBT tags to implement them fully into my project so I'm leaving this issue open and keep hoping for my requested data to be added. :(

**Comment by skyblockz** [comment]
> Pets have been added to the API but only their id, xp and rarity. I still need the pets' NBT tags to implement them fully into my project so I'm leaving this issue open and keep hoping for my requested data to be added. :(

Like what other information you need from them?

**Comment by Flachdachs** [comment]
I guess she means everything that the NBT data usually contain, especially the lore text and skull data.

**Comment by LeaPhant** [comment]
> I guess she means everything that the NBT data usually contain, especially the lore text and skull data.

Yea, I've talked to admins though and apparently it seems unlikely to be added because the current infrastructure doesn't allow it easily. I guess we'll have to gather and calculate all the pet data ourselves unfortunately.
